attribute_assign.py

```diff
@@ -90,9 +90,26 @@ def assign_attribute(stats):
 
     # Deer
     if stats.mount in Deer_list:
-        stats.dex = stats.dex + random.randint(1, 5)
+        stats.dex = stats.dex + random.randint(1, 3)
+    if stats.mount in Adult_Deer_list:
+        stats.str = stats.str + stats.str + 1
     if stats.mount in Majestic_Deer_list:
-        stats.str = stats.str + random.randint(1, 5)
+        stats.str = stats.str + random.randint(1, 3)
+    if stats.mount in Vines_Deer_list:
+        stats.wis = stats.wis + 1
+    if stats.mount in Slime_Deer_list:
+        stats.dex = stats.dex - random.randint(1, 3)
+        stats.luc = stats.luc + random.randint(1, 3)
+    if stats.mount in Bandana_purple_Deer_list:
+        stats.con = stats.con + 1
+    if stats.mount in Bandana_red_Deer_list:
+        stats.int = stats.int + random.randint(1, 2)
+    if stats.mount in One_Blink_Deer_list:
+        stats.cha = stats.cha + random.randint(0, 1)
+    if stats.mount in Two_Blink_Deer_list:
+        stats.cha = stats.cha + stats.cha + 1
```

Comment:
I think this might be doubling CHA

Reply:
No deer in One_Blink_Deer_List are also on Two_Blink_Deer_List if it's that what you mean. It's very possible that i used some formula in the wrong way

Reply:
Yeah so this line 110: 
stats.cha = stats.cha + stats.cha + 1

So if CHA is 12 then;
12 + 12 + 1 = 25

Can probably just remove one of the stats.cha after the equals sign :)

Reply:
Should i do the same on all the lines that use this formula? LIke 110 114 118

Reply:
Wait is it already edited? Because I was confused not seeing it in the files now

Reply:
All good! looks like your file replacement commit has this covered :)